### PR TITLE
Add documentation/usage instructions

### PR DESCRIPTION
**Pull request by ntBre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,201 @@
 # curato
 Chemical dataset curation
+
+## Usage
+There are three main phases to using cura, each of which is described in its own
```
Comment (jaclark5) [inline README.md:5]:
```suggestion
There are three main phases to using curato - store, query, and serve - each of which is described in its own
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,201 @@
...
+section below. I never got around to making cura a proper package that you can
```
Comment (jaclark5) [inline README.md:6]: Should instances of `cura` actually be `curato`?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,201 @@
...
+install and use directly in a conda environment, so current operation basically
+assumes that you're working within the repo.
+
+### Storing data in the database
+The first cura subcommand is `store`, currently invoked with something like
+```shell
+python cura/store.py FILENAME [-n NPROCS] [-t TOTAL]`
+```
+
+`FILENAME` is the required name of a file containing a list of SMILES to load
```
Comment (jaclark5) [inline README.md:16]: Does that mean a text file with one smiles per line, or does a csv work? What format?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,201 @@
...
+into the database. The optional `NPROCS` sets the size of the
+`multiprocessing.Pool` to use for parallelizing the molecule processing, and
+`TOTAL` is an optional total number of lines used for progress reporting[^1].
```
Comment (jaclark5) [inline README.md:19]: So it only shows you the last `TOTAL` number of lines of output? As a user, if I don't know how much output there is, I wouldn't know how much output is expected and I wouldn't want to miss a warning.

Or is this a number of lines in an sql table? Like setting the number of values printed in numpy output from a large array?

File: figs/index.png
Comment (jaclark5) [inline figs/index.png:1]: What is the dataset number?

**Review by jaclark5 — COMMENTED** [review]
Very cool, I'd love to talk to you about this a little more.

File: README.md (hunk 4 above)
Comment (ntBre) [reply README.md:19]: No, it's used by tqdm for progress reporting. Just leave it out if you don't know it, or run `wc -l` on the file and provide that.

File: figs/index.png
Comment (ntBre) [reply figs/index.png:1]: It's the number of molecules in the dataset.

File: README.md (hunk 2 above)
Comment (ntBre) [reply README.md:6]: No, the repo is curato, but the python package is still called cura.

File: README.md (hunk 3 above)
Comment (ntBre) [reply README.md:16]: One SMILES per line

**Pull request merged**